optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -155,6 +162,49 @@ class AddBlurryImagePlaceholders {
     });
     return false;
   }
+
+  /**
+   * Checks if an image should have a blurred image placeholder.
+   * @param {Node} node The DOM element that is being checked to see if it
+   * should have a blurred placeholder.
+   * @param {string} src The image source that is being checked.
+   * @param {string} tagName The type of element that is being checked.
+   * @return {boolean} Whether or not the element should have a blurred
+   * placeholder child.
+   * @private
+   */
+  passesHueristic_(node, src, tagName) {
+    // Checks to see if the image is a jpeg.
+    if (!src.endsWith('.jpg') && !src.endsWith('jpeg')) {
+      return false;
+    }
+
+    // Checks if the image is a poster or a responsive image
```

Comment:
Is `responsive` a good heuristic? I'm thinking of product listing pages with many images with a fixed dimension (similar to google image search).

Reply:
Most product listing pages probably use `amp-list` which won't benefit, though I can see image galleries making use of this. Would love to hear your thoughts on other applications.

There's also an outstanding Firefox performance issue with CSS blur which is why we're keeping the placeholder limit low.

Reply:
Thought about this again. I agree, in most cases you'd still be using layout=responsive for displaying a grid of images.